api-samples/tabs/inspector/tabs_api.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
```

Comment:
Could we maybe rename all of the tabs_api files to something better, e.g window_and_tabs_manager.html?